src/ddmd/semantic.d

```diff
@@ -60,9 +59,8 @@ extern(C++) Expression semantic(Expression e, Scope* sc)
  */
 extern(C++) Initializer semantic(Initializer init, Scope* sc, Type t, NeedInterpret needInterpret)
 {
-    scope v = new InitializerSemanticVisitor(sc, t, needInterpret);
-    init.accept(v);
-    return v.result;
+    import ddmd.initsem;
```

Comment:
Should probably explicitly import only `initializerSemantic` here.

Reply:
I plan on deleting that function when this initiative is complete anyway.